DependencyInjection/DizdaCloudBackupExtension.php

```diff
@@ -27,6 +27,15 @@ public function load(array $configs, ContainerBuilder $container)
         $loader->load('services.yml');
 
         /* Config output file */
+        if(isset($config['error_notification'])){
+            $container->setParameter('dizda_cloud_backup.error_notification', true);
```

Comment:
This line and below are not necessaries, because all nodes are saved into the array parameter `dizda_cloud_backup.error_notification`, so you can get them with `$this->getContainer()->getParameter('dizda_cloud_backup.error_notification')`